internal/graph/check_fast_path.go

```diff
@@ -528,10 +528,7 @@ ConsumerLoop:
 	return res, lastErr
 }
 
-func constructLeftChannels(ctx context.Context,
-	req *ResolveCheckRequest,
-	relationReferences []*openfgav1.RelationReference,
-	relationFunc checkutil.V2RelationFunc) ([]chan *iterator.Msg, error) {
+func constructLeftChannels(ctx context.Context, req *ResolveCheckRequest, relationReferences []*openfgav1.RelationReference, relationFunc checkutil.V2RelationFunc, concurrencyLimit uint32) ([]chan *iterator.Msg, error) {
```

Comment:
nit for readability:

```suggestion
func constructLeftChannels(
	ctx context.Context,
	req *ResolveCheckRequest,
	relationReferences []*openfgav1.RelationReference,
	relationFunc checkutil.V2RelationFunc,
	concurrencyLimit uint32,
) ([]chan *iterator.Msg, error) {
```

Reply:
Done.